test/engine/TextIndexScanForWordTest.cpp

```diff
@@ -73,10 +75,10 @@ TEST(TextIndexScanForWord, WordScanBasic) {
 
   TextIndexScanForWord s2{qec, Variable{"?text1"}, "testing"};
 
-  ASSERT_EQ(s2.getResultWidth(), 1);
+  ASSERT_EQ(s2.getResultWidth(), 2);
 
   result = s2.computeResultOnlyForTesting();
-  ASSERT_EQ(result.idTable().numColumns(), 1);
+  ASSERT_EQ(result.idTable().numColumns(), 2);
```

Comment:
As discussed, let the `getQec()`function take the contents of a `ContextFile`. Then we can finally properly test the test index.

Reply:
the getQec() function now takes the contents of a docsfile and wordsfile as string. This is written to the testing directory as testindexbasename.docsfile (same for wordsfile) to simulate the normal text index building.